moabb/datasets/phmd_ml.py

```diff
@@ -95,7 +95,7 @@ def _get_single_subject_data(self, subject):
         S = data["data"][:, first_channel:last_channel]
         stim = data["data"][:, -1]
 
-        X = np.concatenate([S, stim[:, None]], axis=1).T
+        X = np.concatenate([S * 1e-6, stim[:, None]], axis=1).T
```

Comment:
I think this should go in another PR. This does not seem related to BIDS

Reply:
I will move